app/src/test/java/com/codesoom/assignment/controllers/HelloControllerTest.java

```diff
@@ -0,0 +1,49 @@
+package com.codesoom.assignment.controllers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("HelloController 클래스")
+public final class HelloControllerTest {
+    private HelloController helloController = null;
+
+    @Nested
+    @DisplayName("sayHello 메소드는")
+    class Describe_sayHello {
+        @Nested
+        @DisplayName("선조건")
+        class Context_precondition {
+            @Nested
+            @DisplayName("HelloController가 null인 환경에서")
+            class Context_helloController_null {
+                @Test
+                @DisplayName("NullPointerException을 발생시킨다.")
```

Comment:
```suggestion
                @DisplayName("NullPointerException을 던진다.")
```

`sayHello 메소드는 NPE을 발생시킨다`는 아쉬운 표현입니다. 자바 프로그래밍 언어에서 `throw` 키워드를 어디에 붙이는지 생각해 보세요. 메소드는 예외를 던집니다(throw).

`sayHello(...) throw NullPointerException`

이걸 영어 문장이라 생각하고 읽어봅시다. 그리고 한국어로 번역해서 생각해 봅시다. 이런 의미가 되겠죠.

`sayHello(...)는 NullPointerException을 던진다`